pkgs/servers/pulseaudio/default.nix

```diff
@@ -41,7 +43,8 @@ stdenv.mkDerivation rec {
     sha256 = "11j682g2mn723sz3bh4i44ggq29z053zcggy0glzn63zh9mxdly3";
   };
 
-  patches = [ ./caps-fix.patch ];
+  patches = [ ./caps-fix.patch ]
```

Comment:
Where did this patch come from? It looks like you can probably use `fetchpatch` instead.